File: v2/go/audio/main.go
package main

import (
	"bytes"
	"encoding/base64"
	"encoding/json"
	"fmt"
	"io/ioutil"
	"net/http"
)

func main() {
	url := "http://api-audio-bj.fengkongcloud.com/v2/saas/anti_fraud/audio"
	accessKey := "{ACCESS_KEY}"
	btId := "{BT_ID}"
	filename := "../files/demo.pcm"

	contentBytes, _ := ioutil.ReadFile(filename)
	content := base64.StdEncoding.EncodeToString(contentBytes)
	payload := map[string]interface{}{
		"accessKey": accessKey,
		"type":      "DEFAULT",
		"appId":     "default",
		"btId":      btId,
		"callback":  "https://jsonplaceholder.typicode.com/posts/",
		"data": map[string]interface{}{
			// url 和 content 至少提供一个
			// "url":  "https://jsonplaceholder.typicode.com/audio",
			"content": content,
			"formatInfo": map[string]interface{}{
				"format": "pcm",
				"rate":   8000,
				"track":  1,
			},
		},
	}
	b, _ := json.Marshal(payload)
	resp, _ := http.Post(url, "application/json", bytes.NewBuffer(b))

	var data map[string]interface{}
	if resp != nil {
		respBytes, _ := ioutil.ReadAll(resp.Body)
		_ = json.Unmarshal(respBytes, &data)
		// 通过 code 和 riskLevel 判断返回，具体请参考接口文档
		fmt.Println(data)
	}
}

File: v2/nodejs/audio/main.js
const utils = require('../lib/utils')
const fs = require('fs')

const url = 'http://api-audio-bj.fengkongcloud.com/v2/saas/anti_fraud/audio'
const access_key = '{ACCESS_KEY}'
const bt_id = '{BT_ID}'
const filename = '../files/demo.pcm'

const content = fs.readFileSync(filename, {encoding: 'base64'})
const payload = {
    'accessKey': access_key,
    'type': 'AD_PORN_POLITICAL_MOAN',
    'appId': 'default',
    'btId': bt_id,
    'callback': 'https://jsonplaceholder.typicode.com/posts/',
    'data': {
        // url 和 content 至少提供一个
        // 'url': 'https://jsonplaceholder.typicode.com/audio',
        'content': content,
        'formatInfo': {
            'format': 'pcm',
            'rate': 8000,
            'track': 1,
        },
    },
}

utils.sendRequest(url, 'POST', payload, data => {
    // 通过 code 和 riskLevel 判断返回，具体请参考接口文档
    console.log(data)
})
